src/useIsShake.ts

```diff
@@ -1,18 +1,20 @@
-import React, { useState, useEffect } from 'react';
+import { useState, useEffect } from 'react';
 import { Accelerometer } from 'expo-sensors';
+import { Subscription } from 'expo-sensors/build/Pedometer';
+
+type Poll = number | null;
+type IsShakeProps = { x: number };
 
 export function useIsShake() {
-  const [{ x }, setData] = useState({
```

Comment:
`{x}` value is never read and initial value is not necessary for state value as `setData` is invoked by `Accelerometer` listener and passed an `Accelerometer Measurement` which will then be read.